src/Symfony/Contracts/HttpClient/Exception/RedirectionExceptionInterface.php

```diff
@@ -18,6 +18,6 @@
  *
  * @experimental in 1.1
  */
-interface RedirectionExceptionInterface extends ExceptionInterface
+interface RedirectionExceptionInterface extends ExceptionInterface, HttpExceptionInterface
```

Comment:
`interface RedirectionExceptionInterface extends HttpExceptionInterface`
please do the same `ClientExceptionInterface` and `ServerExceptionInterface`